dds/DCPS/Service_Participant.cpp

```diff
@@ -431,6 +431,15 @@ Service_Participant::get_domain_participant_factory(int &argc,
         }
 
       } else {
+        // Convenient way to run tests in a different place from ini files.
+        const char* const config_dir = ACE_OS::getenv("OPENDDS_TEST_CONFIG_DIR");
```

Comment:
```suggestion
        const char* const config_dir = ACE_OS::getenv("OPENDDS_CONFIG_DIR");
```
Why not make it more general and add to devguide?